rusk/src/lib/http/chain.rs

```diff
@@ -124,4 +127,30 @@ impl RuskNode {
 
         Ok(ResponseData::new(serde_json::to_value(&info)?))
     }
+
+    /// Calculates the average gas price of transactions in the mempool.
+    ///
+    /// It retrieves, at maximum, the top 100 transactions sorted by descending
+    /// order by the gas price, and calculates the average gas price. If there
+    /// are no transactions available in the mempool, it defaults to a gas price
+    /// of 1.
+    async fn get_gas_price(&self) -> anyhow::Result<ResponseData> {
+        let average_gas_price =
+            self.db().read().await.view(|t| -> anyhow::Result<u64> {
+                let (total, count) = t
+                    .get_txs_sorted_by_fee()?
+                    .take(100) // TODO: Figure out a sane default
```

Comment:
Perhaps we should pull the `100` out into a constant, moving the comment on sane defaults with it?

Reply:
> Perhaps we should pull the `100` out into a constant, moving the comment on sane defaults with it?

I'll make it passable per a suggestion by @herr-seppia. This way clients can define their own strategies. 

I'll also add a couple more numbers to deal with outliers, else we'll end up with [Anscombe's quarter](https://en.wikipedia.org/wiki/Anscombe%27s_quartet) situations.